### PR TITLE
Skeleton of searchBooking

### PR DESCRIPTION
**Pull request by amobarak**
# Description and relevant Jira numbers
Just the skeleton of search booking.

## Pull Request checklist

- [ ] [WIP] tag removed from PR title
- [ ] PR has an understandable description

## Git feature branch checklist

- [ ] branch name comply with our branching strategy
- [ ] git branch contains relevant JIRA ticket number
- [ ] branch rebased against the latest develop

## Sign off process checklist

- [ ] Code has been tested manually
- [ ] Tested by QA
- [ ] PO's approval
- [ ] Reviewers selected in Github
- [ ] PR link added to JIRA

### CODE REVIEW

File: src/common/domain/Journal.d.ts [hunk 1]
```diff
@@ -0,0 +1,369 @@
+/* tslint:disable */
```
Comment (TomBillingtonUK) [inline src/common/domain/Journal.d.ts]: should we not be using the common schema on npm here?

Comment (szabikr) [reply src/common/domain/Journal.d.ts]: I agree with Tom, we should bring the common schema from api-definitions

**Review by szabikr — APPROVED** [review]

File: src/common/domain/Journal.d.ts (hunk 1 above)
Comment (amobarak) [reply src/common/domain/Journal.d.ts]: Do you mean the `mes-journal-schema` ?

Comment (TomBillingtonUK) [reply src/common/domain/Journal.d.ts]: yes

**Review by szabikr — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**